components/data_comps/docn/cime_config/config_component.xml

```diff
@@ -176,8 +176,7 @@
       <value compset="(AMIP_|HIST_|20TR_|5505_|PIPD_|%TSCH)" grid="a%0.9x1.25.*_oi%0.9x1.25"	>$DIN_LOC_ROOT/atm/cam/sst/sst_HadOIBl_bc_0.9x1.25_1850_2012_c130411.nc</value>
       <value compset="(AMIP_|HIST_|20TR_|5505_|PIPD_|%TSCH)" grid="a%0.47x0.63.*_oi%0.47x0.63"	>$DIN_LOC_ROOT/atm/cam/sst/sst_HadOIBl_bc_0.47x0.63_1850_2012_c130411.nc</value>
       <value compset="(AMIP_|HIST_|20TR_|5505_|PIPD_|%TSCH)" grid="a%0.23x0.31.*_oi%0.23x0.31"	>$DIN_LOC_ROOT/atm/cam/sst/sst_HadOIBl_bc_0.23x0.31_1850_2012_c130411.nc</value>
-      <value compset="1850_" grid=".+"								>$DIN_LOC_ROOT/atm/cam/sst/sst_HadOIBl_bc_1x1_clim_pi_c101029.nc</value>
-      <value compset="1850S_" grid=".+"								>$DIN_LOC_ROOT/ocn/docn7/SSTDATA/sst_ice_CMIP6_DECK_E3SM_1x1_1850_clim_c20190125.nc</value>
```

Comment:
@mahf708 I'm not sure if I understand your question.  We decided previously to use F2010 for the ERF runs, but we will still have an F1850 compset after this change.

For some background, the "S" in F1850S is a naming convention that refers to a spun-up initial condition.  This initial condition has to be changed when the compset is modified in a way that is climate-changing, and that may be why it was removed here (since we don't have a new initial condition yet).  

If I'm understanding the code changes correctly, the F1850 compset hasn't been removed here, but it's been given a different sea surface temperature file (the one that was previously used for F1850S).

This is likely correct, but I just wanted to double-check that this was an intended change and not accidental.  And, I'm also not sure if removing the F1850S compset was intended or needed.  (Presumably it could be left as-is for now, and then updated once a new initial condition is available).

Reply:
Adding a separate comment to address your question regarding the use of F1850 versus F2010 for the AWG work here.  We previously decided to use F2010 as a baseline, in keeping with prior practice.

We did recently have a discussion with @xuezhengllnl about the idea of additionally calculating ERF using the 1850 compset as a baseline, and I think that is something we should try.  Originally, we assumed that the differences from using an 1850 or 2010 baseline would be small.  However, I now realize that with the new chemistry, we should expect large changes in the oxidative environment between PD and PI with the new chemistry features.  As a result, I expect that we will measure a significantly different value for ERFaer when we use PD as a baseline in any versions where the new chemistry is turned on.  So, I agree that it would be a good idea for us to test this soon, although I thnk it is not extremely urgent.

Additionally, the PI/PD conditions selected for volcanic aerosols could potentially also have an impact on our measurement of ERFaer.  We previously assumed that this impact would be small, given that the volcanic aerosol is confined to the stratosphere and

However, the [analysis Ziming presented recently](https://acme-climate.atlassian.net/wiki/spaces/ATMOS/pages/3769925633/2023-06-02+Aerosol+Working+Group+Meeting+notes) showed that the MAM5 feature causes non-negligible changes in both clouds and clearsky radiation, which also suggests the ERF calculation could be sensitive to the choice of baseline volcanic emissions.  So we may need to pay more attention to this than we previously thought.  I think github is not the best to continue a detailed discussion of this topic, but I will make a note that we should follow up on this for the AWG effort.